guides/03_API_Overview.md

```diff
@@ -0,0 +1,296 @@
+
+### 1a. Create a logger
+
+Create a [ConsoleLogger](https://aws.github.io/amazon-chime-sdk-js/classes/consolelogger.html) to log everything to the browser console. You can also implement the [Logger](https://aws.github.io/amazon-chime-sdk-js/interfaces/logger.html) interface to customize logging behavior.
+
+```
+const logger = new ConsoleLogger(`'MeetingLogs', LogLevel.INFO);`
+```
+
+### 1b. Create a device controller
+
+Create a [DefaultDeviceController](https://aws.github.io/amazon-chime-sdk-js/classes/defaultdevicecontroller.html#constructor) and pass in the Logger you created. This object allows you to enumerate and select audio and video devices and control some of their features, even before the meeting session has been created.
+
+```
+const deviceController = new DefaultDeviceController(logger);
+```
+
+### 1c. Create a meeting session configuration
+
+Create a [MeetingSessionConfiguration](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#constructor) object with the responses to [chime:CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html) and [chime:CreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html). Your server application should make these API calls and securely pass the meeting and attendee responses to the browser client application.
+
+```
+const configuration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
+```
+
+### 1d. Create a meeting session
+
+Using the above objects, create a [DefaultMeetingSession](https://aws.github.io/amazon-chime-sdk-js/classes/defaultmeetingsession.html#constructor).
+
+```
+const meetingSession = new DefaultMeetingSession(configuration, logger, deviceController);
+```
+
+## 2. Configure the session
+
+Before starting the meeting session, you should configure audio and video devices. By default, no devices are selected.
+
+### 2a. Configure the device label trigger (optional)
+
+When obtaining devices to configure, the browser may initially decline to provide device labels due to privacy restrictions. However, without device labels the application user will not be able to select devices by name. When no labels are present, the device-label trigger is run. The default implementation of the device-label trigger requests permission to the mic and camera. If the user grants permission, the device labels will become available.
+
+You can override the behavior of the device-label trigger by calling `meetingSession.audioVideo.[setDeviceLabelTrigger(trigger)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#setdevicelabeltrigger)`.
```

Comment:
Need to remove backticks. GitHub Markdown is not able to render inline code with embedded URLs.

Reply:
Ok, I updated to remove the backticks. I also found that <audio> and <video> didn't render, so I removed `<` and `>` as well.

Reply:
Addressing in https://github.com/aws/amazon-chime-sdk-js/pull/243 since this got merged.